utils/layout/util.js

```diff
@@ -0,0 +1,120 @@
+export const validBlockTypes = [
+	'pubs',
+	'text',
+	'html',
+	'banner',
+	'pages', // TODO(ian): Remove this after migration
+	'collections-pages',
+];
+
+export const getPubsByBlockIndex = (blocks, pubs) => {
```

Comment:
This is `generateRenderLists` by a more descriptive name and moved from `utils/pages.js`